applications/FluidDynamicsApplication/symbolic_generation/navier_stokes/symbolic_stokes_cpp_template.cpp

```diff
@@ -0,0 +1,369 @@
+    SymbolicStokesData<2,3>& rData,
+    VectorType& rRHS)
+{
+    const double rho = rData.Density;
+    const double mu = rData.EffectiveViscosity;
+
+    const double h = rData.ElementSize;
+    const double dt = rData.DeltaTime;
+    const double bdf0 = rData.bdf0;
+    const double bdf1 = rData.bdf1;
+    const double bdf2 = rData.bdf2;
+
+    const double dyn_tau = rData.DynamicTau;
+
+    const auto& v = rData.Velocity;
+    const auto& vn = rData.Velocity_OldStep1;
+    const auto& vnn = rData.Velocity_OldStep2;
+    const auto& f = rData.BodyForce;
+    const auto& p = rData.Pressure;
+    const auto& stress = rData.ShearStress;
+
+    // Get shape function values
+    const auto& N = rData.N;
+    const auto& DN = rData.DN_DX;
+
+    // Stabilization parameters
+    constexpr double stab_c1 = 4.0;
+    constexpr double stab_c2 = 2.0;
+    const array_1d<double, 2> v_aux = prod(trans(rData.Velocity), N);
+
+    auto& rhs = rData.rhs;
+
+    //substitute_rhs_2D
+
+    noalias(rRHS) += rData.Weight * rhs;
+}
+
+template <>
+void SymbolicStokes<SymbolicStokesData<3,4>>::ComputeGaussPointRHSContribution(
+    SymbolicStokesData<3,4>& rData,
+    VectorType& rRHS)
+{
+    const double rho = rData.Density;
+    const double mu = rData.EffectiveViscosity;
+
+    const double h = rData.ElementSize;
+    const double dt = rData.DeltaTime;
+    const double bdf0 = rData.bdf0;
+    const double bdf1 = rData.bdf1;
+    const double bdf2 = rData.bdf2;
+
+    const double dyn_tau = rData.DynamicTau;
+
+    const auto& v = rData.Velocity;
+    const auto& vn = rData.Velocity_OldStep1;
+    const auto& vnn = rData.Velocity_OldStep2;
+    const auto& f = rData.BodyForce;
+    const auto& p = rData.Pressure;
+    const auto& stress = rData.ShearStress;
+
+    // Get shape function values
+    const auto& N = rData.N;
+    const auto& DN = rData.DN_DX;
+
+    // Stabilization parameters
+    constexpr double stab_c1 = 4.0;
+    constexpr double stab_c2 = 2.0;
+    const array_1d<double, 3> v_aux = prod(trans(rData.Velocity), N);
+
+    auto& rhs = rData.rhs;
+
+    //substitute_rhs_3D
+
+    noalias(rRHS) += rData.Weight * rhs;
+}
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+// Private serialization
+
+template< class TElementData >
+void SymbolicStokes<TElementData>::save(Serializer& rSerializer) const
+{
+    using BaseType = FluidElement<TElementData>;
+    KRATOS_SERIALIZE_SAVE_BASE_CLASS(rSerializer, BaseType );
+}
+
+
+template< class TElementData >
+void SymbolicStokes<TElementData>::load(Serializer& rSerializer)
+{
+    using BaseType = FluidElement<TElementData>;
+    KRATOS_SERIALIZE_LOAD_BASE_CLASS(rSerializer, BaseType);
+}
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+// Class template instantiation
+
+template class SymbolicStokes< SymbolicStokesData<2,3> >;
+template class SymbolicStokes< SymbolicStokesData<3,4> >;
+
+}
```

Comment:
Probably a new line here?

Reply:
mmm... I think I do not get your point at all...

Reply:
It is recommended to end files with a new line. 

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

For me, its just a little painful to watch : P

Reply:
:open_mouth: I wasn't aware of this... I will add it. Thanks guys!

Reply:
Newline added.